app/scripts/services/quota.js

```diff
@@ -277,6 +282,45 @@ angular.module("openshiftConsole")
         return _.some(quotas, isExceeded) || _.some(clusterQuotas, isExceeded);
     };
 
+    //Specific to storage quotas for max claim count and max storage usage
+    // and the amount of claims and storage already used
+
+    var getStorageQuotas = function(quotas) {
+      // Currently, there can be multiple quotas per namespace (bug?)
```

Comment:
nope, intentional. as many quotas as you want, plus you have to deal with clusterresourcequotas.